### DISCUSSION TITLE
Conditional classes don't work with tailwind

### DISCUSSION THREAD

**Discussion by steckes** [leptos-rs/leptos#1527] [Q&A] [answered]
When I add `class:text-6xl=true` to a div the class is getting applied to the object, but the css code for this class is not copied by tailwind.
A workaround is to use `class="text-6xl"` in an empty object so the css code is getting copied. Is there a solution to this? 

I only checked client side rendering with the trunk template from the examples directory. Is this happening as well when using cargo-leptos?

**Comment by gbj** [comment, marked as answer]
Does it work if you do `class=("text-6xl", true)`? In any case it will just be a matter of needing to tweak the regexes that Tailwind is scanning your files for, but I'm not very experienced with Tailwind.

**Reply by steckes** [reply]
Cool that is working, thanks 👍
